src/components/DataTable/DataTable.less

```diff
@@ -0,0 +1,14 @@
+.lucid-DataTable {
+	.lucid-Table-density-extended {
+		.lucid-Table-Tbody {
+			.lucid-Table-Tr {
+				.lucid-Table-Td.lucid-DataTable-empty-cell {
```

Comment:
I'm not sure all these layers are necessary. Also, if we move this into `Table` it might fit better over there.